### ISSUE TITLE
Bug: trying to add a EPG obtain error because is GTM+2

### ISSUE THREAD

**Issue by Crosus97** [SenexCrenshaw/StreamMaster#264]
**Describe the Bug**
When you try to add a Spanish Know EPG [DobleEPG](https://github.com/davidmuma/EPG_dobleM) (General or Emby in my case) show the error:
![image](https://github.com/user-attachments/assets/87bc4130-dc10-4c63-b428-92572016396b)

This error doesnt occur on the "Old" version of Streammaster

I think issue #203  has the same error but does not provide enough information on the issue 

**Stream Master Info**
- [channels-0.15.0.60](https://hub.docker.com/layers/senexcrenshaw/streammaster/channels-0.15.0.60/images/sha256-18a7dee37514ae09682b1ac796ccb555604c57d170260dfbfb242a16da8f5cdf?context=explore)

**Link to logs (required)**
- [Logs](https://gist.github.com/Crosus97/808eef4b2b784719a4eafa4992ba0c56)

**Comment by DieguitoCigala** [comment]
I have the same problem and i use the epg without +0200 i receive the next message "String '202410270000' was not recognized as a valid DateTime"

**Comment by DieguitoCigala** [comment]
Now its OK. Thanks friend!

**Comment by Crosus97** [comment]
Perfect, this work perfectly 

TY so much for your Work

**Comment by Crosus97** [comment]
![image](https://github.com/user-attachments/assets/9daa9c95-ad67-4d6c-8e03-b07a45d631bc)


i obtain a new error, in this case because is GTM+1 Maybe?¿

**Comment by DieguitoCigala** [comment]
Yes, the same error with +01 "System.FormatException: The DateTime represented by the string '20241032010000 +01:00' is not supported in calendar 'System.Globalization.GregorianCalendar'."

**Comment by SenexCrenshaw** [comment]
gotta love dates like
20241032010000
32nd day of october!

**Comment by SenexCrenshaw** [comment]
try 
``0.17.3``
tested with
["20240824240000 +0200", "202410270000", "20241032010000 +01:00", "20241102243000 +0200", "2024040830000 +0000"]

**Comment by DieguitoCigala** [comment]
Hi Senex, I think that +0100 and +0200 only indicate GMT+1 or GMT+2 but they do not have to be added to the actual time of the date.

**Comment by Crosus97** [comment]
for me its fixed

**Comment by DieguitoCigala** [comment]
Yes, it's fixed. But the schedule appears to me with an hour delay. I think it's because you add +0100 to the program time.

Another issue would be to be able to make this epg work (https://raw.githubusercontent.com/davidmuma/EPG_dobleM/refs/heads/master/guiaiptv0.xml) that doesn't have +0100 or +0200.

**Comment by DieguitoCigala** [comment]
This is what comes out of my epg provider
  <programme start="20241107080000 +0100" stop="20241107100000 +0100" channel="M+ Clásicos HD">
    <title lang="es">Un hombre para la eternidad</title>
    <desc lang="es">Cine/Drama | 1966 | TP | ★8.0/10
    ...

And this is what comes out of the Stream Master epg
.</desc><date>19700101</date><language>es</language><episode-num system="original-air-date">2024-11-07</episode-num><new /></programme><programme start="20241107100000 +0000" stop="20241107114600 +0000" channel="20"><title>Ocho sentencias de muerte</title><desc>Cine/Comedia | 1949 | ⑯ | ★8.0/10

**Comment by SenexCrenshaw** [comment]
i enabled negative values in the next pr.
